Spigot-API-Patches/0039-LootTable-API.patch

```diff
@@ -0,0 +1,207 @@
+From d33eb4f175086d7ec75ff3e3893a658788a3311b Mon Sep 17 00:00:00 2001
+From: Aikar <aikar@aikar.co>
+Date: Sun, 1 May 2016 15:19:49 -0400
+Subject: [PATCH] LootTable API
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/loottable/Lootable.java b/src/main/java/com/destroystokyo/paper/loottable/Lootable.java
+new file mode 100644
+index 0000000..61cc5dc
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/loottable/Lootable.java
+@@ -0,0 +1,88 @@
++package com.destroystokyo.paper.loottable;
++
++import org.bukkit.block.Block;
++
++import java.util.UUID;
++
++public interface Lootable {
```

Comment:
To be honest, it wouldn't be a bad idea to see about mimicking the serializable conditions and definitions that Minecraft already uses for these things in the API.